modules/luci-mod-network/htdocs/luci-static/resources/view/network/dhcp.js

```diff
@@ -841,16 +859,53 @@ return view.extend({
 			so.value(ipv4, ipaddrs[ipv4] ? '%s (%s)'.format(ipv4, ipaddrs[ipv4]) : ipv4);
 		});
 
-		so = ss.option(form.Value, 'leasetime', _('Lease time'));
+		so = ss.option(form.Value, 'leasetime', 
+			_('Lease time'),
+			_('Host-specific lease time, e.g. <code>5m</code>, <code>3h</code>, <code>7d</code>.'));
 		so.rmempty = true;
-
-		so = ss.option(form.Value, 'duid', _('DUID'));
+		so.value('5m', _('5m (5 minutes)'));
+		so.value('3h', _('3h (3 hours)'));
+		so.value('12h', _('12h (12 hours - default)'));
+		so.value('7d', _('7d (7 days)'));
+		so.value('infinite', _('infinite (lease does not expire)'));
+
+		so = ss.option(form.Value, 'duid',
+			_('DUID'),
+			_('The DHCPv6-DUID of this host.'));
 		so.datatype = 'and(rangelength(20,36),hexstring)';
 		Object.keys(duids).forEach(function(duid) {
 			so.value(duid, '%s (%s)'.format(duid, duids[duid].hostname || duids[duid].macaddr || duids[duid].ip6addr || '?'));
 		});
 
-		so = ss.option(form.Value, 'hostid', _('IPv6 suffix (hex)'));
+		so = ss.option(form.Value, 'hostid',
+			_('IPv6-Suffix (hex)'),
+			_('The IPv6 interface identifier (address suffix) as hexadecimal number (max. 8 chars).'));
+		so.datatype = 'and(rangelength(0,8),hexstring)';
+
+		so = ss.option(form.Value, 'tag',
+			_('Tag'),
+			_('Set the given tag (used to match hosts).'));
+		so.value('known');
+		so.value('!known', _('!known (not known)'));
```

Comment:
That looks more like a tag to match, not a tag name to set. Is this `!known` choice really correct?

Reply:
As used in the dnsmasq man page:
https://thekelleys.org.uk/dnsmasq/docs/dnsmasq-man.html


The set:<tag> construct sets the tag whenever this --dhcp-host directive is in use. This can be used to selectively send DHCP options just for this host. More than one tag can be set in a --dhcp-host directive (but not in other places where "set:<tag>" is allowed). When a host matches any --dhcp-host directive (or one implied by /etc/ethers) then the special tag "known" is set. This allows dnsmasq to be configured to ignore requests from unknown machines using --dhcp-ignore=tag:!known If the host matches only a --dhcp-host directive which cannot be used because it specifies an address on different subnet, the tag "known-othernet" is set.

Reply:
Yeah, so this option is about *setting* a tag for the host, right? At least the description says so. I understand the man page you quoted in a way that "known", "!known", "other-network" are tag values set by dnsmasq itself which you can use to match but the way I understand it, it makes little sense to set those values manually, so they shouldn't be in the default value suggestions.

Unless the uci "set" option is for *matching* tags - in this case the description would be wrong

Reply:
You're right - split it into the two available fields.